README.md

```diff
@@ -3,7 +3,7 @@
 Login by Auth0
 ====
 
-Single Sign On for Enterprises + Social Login + User/Passwords. For all your WordPress instances. Powered by Auth0.
+Single Sign On for Enterprises + Social Login + User/Passwords + Passwordless. For all your WordPress instances. Powered by Auth0.
```

Comment:
Might be nicer as a bullet point list vs +

Reply:
I agree since this is getting bigger, but he's already using a bulleted list below this paragraph.

Reply:
How about:

"SSO, Social Login, Username & Passwords,  Passwordless, and more."

Reply:
Is there more ? ;)